.murdock

```diff
@@ -365,14 +365,22 @@ compile() {
         fi
     fi
 
+    # log some build stats
     if [ -d ${BINDIR} ]
     then
-        echo "-- build directory size: $(du -sh ${BINDIR} | cut -f1)"
-
+    echo -n "{\"build\": { \"bin/\": $(du -sh ${BINDIR} | cut -f1)}}"
```

Comment:
```suggestion
        echo -n "{\"build\": { \"bin/\": $(du -sh ${BINDIR} | cut -f1)}}"
```

Reply:
yes, thanks. also, the "-n" is a leftover from me doing multiple lines.

(it was next in line, now there are *six* ahead... hehe)